docs/enhancements/backup-restore-support.md

```diff
@@ -0,0 +1,151 @@
+---
+title: backup-restore-support
+authors:
+  - "@danielerez"
+creation-date: 2024-08-08
+last-updated: 2024-08-08
+---
+
+# Backup/Restore Support
+
+## Summary
+
+The Assisted Installer should support backup/restore and disaster recovery scenarios, either using OADP (OpenShift API for Data Protection) for ACM (Advanced Cluster Management), or, using ZTP (Zero Touch Provisioning) flows.
```

Comment:
By "using ZTP", do you mean copying the relevant resources used by ZTP in any way?  (e.g., gitops)

Reply:
I think OADP and the ACM DR solution generally are totally unaware of if someone is using gitops or not so we just need to make it work in both cases.

Or are you suggesting gitops is an alternative to OADP in some way?